### PR TITLE
Remove caas feature flag.

### PR DESCRIPTION
**Pull request by ExternalReality**
## Description of change

Remove caas feature flag.

Why is this change needed?

CAAS features should be available by default. 

## QA steps

Tests should still work.

### CODE REVIEW

File: apiserver/allfacades.go [hunk 1]
```diff
@@ -151,15 +151,14 @@ func AllFacades() *facade.Registry {
...
-		// Move these to the correct place above once the feature flag disappears.
-		reg("CAASFirewaller", 1, caasfirewaller.NewStateFacade)
-		reg("CAASOperator", 1, caasoperator.NewStateFacade)
-		reg("CAASAgent", 1, caasagent.NewStateFacade)
-		reg("CAASOperatorProvisioner", 1, caasoperatorprovisioner.NewStateCAASOperatorProvisionerAPI)
-		reg("CAASUnitProvisioner", 1, caasunitprovisioner.NewStateFacade)
-	}
+
+	// CAAS related facades.
+	// Move these to the correct place above once the feature flag disappears.
```
Comment (ExternalReality) [inline apiserver/allfacades.go]: What is the correct place that is being referenced here?

**Comment by ExternalReality** [comment]
[Remove CAAS feature flag](https://trello.com/c/zMKZ2zCH/148-remove-caas-feature-flag)

**Comment by ExternalReality** [comment]
!!build!!

File: state/model.go [hunk 2]
```diff
@@ -281,7 +279,7 @@ func (m ModelArgs) Validate() error {
 	if m.Type == modelTypeNone {
 		return errors.NotValidf("empty Type")
 	}
-	if m.Type == ModelTypeCAAS && !featureflag.Enabled(feature.CAAS) {
+	if m.Type == ModelTypeCAAS {
```
Comment (manadart) [inline state/model.go]: I think the whole conditional should go away.
This would make CAAS types invalid always...

**Comment by ExternalReality** [comment]
$$MERGE$$

**Pull request merged**